lib/src/ffmpeg/ffmpeg_command.dart

```diff
@@ -137,7 +137,7 @@ class CliArg {
   final String name;
   final String? value;
 
-  String toCli() => '-$name ${(value != null) ?  value : ""}';
+  String toCli() => '-$name ${(value != null) ? value : ""}';
```

Comment:
Please add appropriate tests.

Reply:
This was an auto-formatting change, I should revert this.